elrond-wasm/src/types/managed/managed_vec_item.rs

```diff
@@ -0,0 +1,69 @@
+use elrond_codec::{NestedDecode, NestedEncode};
+
+use crate::{
+    abi::TypeAbi,
+    api::{Handle, ManagedTypeApi},
+};
+
+use super::{BigInt, BigUint, ManagedBuffer, ManagedType};
+
+/// Types that implement this trait can fir inside a `ManagedVec`.
```

Comment:
fir -> fit, though I'm not entirely sure the wording is correct even so. Maybe `Types that implement this trait can be stored inside a ManagedVec`?